src/tools/measure.rs

```diff
@@ -5,7 +5,7 @@ use druid::{Data, Env, EventCtx, PaintCtx};
 use crate::design_space::DPoint;
 use crate::edit_session::EditSession;
 use crate::mouse::{Drag, Mouse, MouseDelegate, TaggedEvent};
-use crate::tools::{EditType, Tool};
+use crate::tools::{axis_locked_point, EditType, Tool};
```

Comment:
I believe the convention is to not import free functions directly, but to call them through the containing module, e.g as `tools::axis_locked_point`?